src/s_main.c

```diff
@@ -1518,7 +1518,7 @@ static void namelist2gui(const char*name, t_namelist*namelist)
     }
     pdgui_vmess("set", "rS",
               name,
-              i, names);
+              (int)i, names);
```

Comment:
good catch.

but i think the fix is wrong. instead, `i` should just be made an `int` (which I did with a8a245d3752af4026a17bb2c1ecb5d7c1a18606e)